src/allTasks/index.js

```diff
@@ -26,79 +27,79 @@ class Task_Index extends Component {
     return (
       <main>
         <section>
-          <h1 className="greeting">
+          <h1 className="greeting"> {/*from App.css*/}
```

Comment:
What is that?? Is this all your new styling stuff??

Reply:
These are just notes as to where to find that class if changes need to be made. I was planning to keep App.css still imported in the index.html file, but I wanted to keep the smaller, more specific styles in their project folders. These are just notes that these particular classes are from App.css. There might be a better way of doing this though...

Reply:
Cool! Thanks :-)